### PR TITLE
Upload page replacement algorithm for fifo #7

### LINKED ISSUE #7: Function for FIFO
This function should accept two inputs (frames: number, referenceString: array).
It should return the number of page faults generated according to FIFO page replacement algorithm.

### PR DESCRIPTION
**Pull request by masterYoda8**

### CODE REVIEW

**Review by fabibec — COMMENTED** [review]
The results are correct, however I'm not sure if the list order is correct at all times, where all the frames should move one position to the right if a new one is inserted. Therefore, I want to propose an alternative solution for you to check out:
```python
def fifo2(frames, reference_string):
    from collections import deque
    frame_list = deque([None]*frames,maxlen=frames)
    page_faults = 0
    for i in reference_string:
        if i not in frame_list:
            # Catch the removed frame if needed
            out = frame_list.pop()
            frame_list.appendleft(i)
            page_faults += 1
        print(frame_list)   
    return page_faults
```

**Review by fabibec — APPROVED** [review]
Nice!

File: fifo.py
Comment (xgsngxguy) [inline fifo.py:1]: simple but functional, good proposal from @fabibec 

in bavaria we say: 'ausgfuxts buerschal'

**Review by xgsngxguy — APPROVED** [review]

**Review by xgsngxguy — CHANGES_REQUESTED** [review]
please merge in dev

**Pull request merged**